Lib/compositor/font.py

```diff
@@ -6,6 +6,17 @@
 from compositor.error import CompositorError
 
 
+def _nameToString(nameRecord):
+    encoding = nameRecord.getEncoding()
+    try:
+        # python 3
+        ret = nameRecord.string.decode(encoding)
```

Comment:
yes, in Python 2 you want bytes. The `string` attribute is usually encoded as a bytes string. When you call `toUnicode`, it gets decoded to a unicode string using the encoding specified in the name record. However, when you later call `tostr`, the unicode string is left untouched in Python 3, but it is re-encoded to bytes, this time using the default ascii encoding (not the one in which the original string was encoded).